### ISSUE TITLE
A11y_AzureDataStudioSQL_Project_CreateNewProject_ScreenReader: NVDA announces incorrect label for "Name" edit field when navigating using tab key.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#14303]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudioSQL_Feb2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG3.3.2;#JAWS;#NVDA;#STP;#AINotInScope;#Element:Edit Field;

### Environment Details:
Application Name: Azure Data Studio
Version: 1.26.0-insider (user setup)
Windows Version: Windows10
Screen reader: NVDA(2020.2), Jaws(2020.2008.24)

### Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Tab till SQL Database Project Home Screen with heading "Projects".
3. Tab till "Create New Project" button or "+" button and hit enter.
5. Tab till "Name" edit field and check what nvda announces when navigating using tab key.

### Actual:
NVDA announces incorrect label "Name. Please fill out this field" for the edit box when navigating using tab key.
Issue is also with "Location edit field"

Same issue is also observed with Jaws

Similar issue is observed in below scenarios:

- Open Existing Project- Project
- Open Existing Project - Workspace

### Expected:
NVDA should announce correct label for edit field which is "Name" only when navigating using tab key.

### User Impact:
Screen reader user will not get correct information about the edit field

**Comment by udeeshagautam** [comment]
@Shivamd1, it was suggested to us in a previous testing round that screen reader should read the message for the field if field is empty. Here is the older bug for the same : https://github.com/microsoft/azuredatastudio/issues/6846

Based on the previous bug, we added the message. Hence this behavior seems by design as per the accessibility rules. Can you please check with the SMEs and clarify the right behavior here.

**Comment by Shivamd1** [comment]
As the message gives the additional information along with label so closing the bug.

**Comment by Yashi1201** [comment]
As per above comment,"As the message gives the additional information along with label so closing the bug."
**GithubTags**:#Closed;
